adrs/Enums.md

```diff
@@ -0,0 +1,25 @@
+# Enum as string in the database
```

Comment:
```suggestion
# Storing the book language in a database
```

Reply:
and I would suggest giving the file a similar name. Right now the title contains the solution to the problem, the outcome of the ADR, not the problem that the ADR solves.